pyedr/pyedr/tests/data/mocks/edr_mock.py

```diff
@@ -0,0 +1,146 @@
+# -*- coding: utf-8 -*-
+import xdrlib
```

Comment:
Could you explain this file? It's not immediately clear to me what it does

Reply:
This is just some helper script to fudge some artificial EDR files with special values. As I'm not aware how to reach some of the code branches with data generated by Gromacs itself, I used this to generate the mocks.
The inclusion of the script is only for archiving/posterity.

Reply:
Added some comments in the file to document its usage a bit more.